### DISCUSSION TITLE
using two different font sizes in a single multi_cell

### DISCUSSION THREAD

**Discussion by Nithyashri-E** [py-pdf/fpdf2#793] [Q&A] [answered]
The following is my requirement:
"ABC" with font size as 10 and "XYZ" with font size 6. I want both "ABC" and "XYZ" in a single multi_cell cell. 
Is there any method that can satisfy this requirement in fpdf? I'm ready to even try alternative methods in case there isn't any straightforward method that can meet this requirement. Kindly let me know whatever it is.

**Comment by Lucas-C** [comment]
Hi @Nithyashri-E!

Very recently we already had a discussion about varying font sizes in https://github.com/PyFPDF/fpdf2/issues/786
But you also want to use the same `multi_cell`...

This is currently not supported, but is technically doable,
and would require minimal changes to `styled_text_fragments` & `_render_styled_text_line()`.

@gmischler is currently working on refactoring this part of `fpdf2` code though,
and I would prefer to wait for his contribution before creating more public interfaces (methods & parameters)
related to text rendering.

**Comment by gmischler** [comment, marked as answer]
@Nithyashri-E, I've just checked in #897, which should solve your problem.
Well, technically it's not going to be a `multi_cell` but an instance of `text_columns()`, and if you need borders around it you'll have to draw them yourself. but you can freely change fonts within line wrapped text.
<s>You can install the currently unreleased code (about to be released as 2.7.6 very soon) like this:

`pip install git+https://github.com/py-pdf/fpdf2.git@master`

and test it in your environment.</s>

Edit: Published with release 2.7.6
